osu.Game.Rulesets.Osu/Objects/Slider.cs

```diff
@@ -73,6 +74,7 @@ public override void ApplyDefaults(ControlPointInfo controlPointInfo, BeatmapDif
 
             Velocity = scoringDistance / timingPoint.BeatLength;
             TickDistance = scoringDistance / difficulty.SliderTickRate;
+            BouncerDistance = Distance;

```

Comment:
Is this needed? Since it's only used from within `Slider`, can we just use `Distance` here?